test/ssl-tests/18-dtls-renegotiate.cnf.dat

```diff
@@ -19,9 +19,8 @@ our $fips_mode;
 
 our @tests = ();
 
-foreach my $sctp ("No", "Yes")
+foreach my $sctp ("No")
```

Comment:
`$sctp` isn't really used anymore and the whole thing isn't a loop anymore.

You could break the loop into two simple if statements. One here:
```perl
if  (!$fips_mode || !disabled("dtls1_2")) {
```
(order to match 16-dtls-certstatus.cnf.dat)